service/common/src/main/java/org/apache/polaris/service/persistence/InMemoryPolarisMetaStoreManagerFactory.java

```diff
@@ -89,12 +89,12 @@ public synchronized Supplier<TransactionalPersistence> getOrCreateSessionSupplie
       RealmContext realmContext) {
     String realmId = realmContext.getRealmIdentifier();
     if (!bootstrappedRealms.contains(realmId)) {
-      bootstrapRealmsAndPrintCredentials(List.of(realmId));
+      bootstrapRealmsFromEnvironment(List.of(realmId));
     }
     return super.getOrCreateSessionSupplier(realmContext);
   }
 
-  private void bootstrapRealmsAndPrintCredentials(List<String> realms) {
+  private void bootstrapRealmsFromEnvironment(List<String> realms) {
```

Comment:
Just renamed this method to better reflect its new purpose (it changed recently).